chord_metadata_service/chord/ingest/views.py

```diff
@@ -93,3 +85,42 @@ def ingest(request):
     # TODO: Schema validation
     # TODO: Rollback in case of failures
     return Response(status=204)
+
+
+@api_view(["POST"])
+@permission_classes([AllowAny])
+def ingest_dataset(request, dataset_id: str, data_type: str):
```

Comment:
misleading function name i think - it isn't ingesting a dataset, it's ingesting into a dataset